hooked/views.py

```diff
@@ -63,7 +64,7 @@ def check_webhook_signature(self, request):
         match we just log an error.)
         """
         app_key = getattr(settings, APP_KEY_SETTING)
-        fail_on_mismatch = getattr(settings, FAIL_ON_MISMATCH_SETTING, False)
+        fail_on_mismatch = getattr(settings, FAIL_ON_MISMATCH_SETTING, True)
```

Comment:
I've updated the README as well now.  I agree that we should confirm everything is working as we expect before defaulting to Fail (it seems to be)